app/src/main/java/com/google/maps/android/compose/StreetViewActivity.kt

```diff
@@ -0,0 +1,100 @@
+package com.google.maps.android.compose
+
+import android.os.Bundle
+import android.util.Log
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.material.Button
+import androidx.compose.material.Switch
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.runtime.snapshotFlow
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.LineHeightStyle
+import androidx.compose.ui.unit.dp
+import com.google.android.gms.maps.StreetViewPanoramaOptions
+import com.google.maps.android.compose.streetview.StreetView
+import com.google.maps.android.compose.streetview.rememberStreetViewCameraPositionState
+import com.google.maps.android.ktx.MapsExperimentalFeature
+
+class StreetViewActivity : ComponentActivity() {
+
+    private val TAG = StreetViewActivity::class.java.simpleName
+
+    @OptIn(MapsExperimentalFeature::class)
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            var isPanningEnabled by remember { mutableStateOf(false) }
+            var isZoomEnabled by remember { mutableStateOf(false) }
+            val camera = rememberStreetViewCameraPositionState()
+            LaunchedEffect(camera) {
+                snapshotFlow { camera.panoramaCamera }
+                    .collect {
+                        Log.d(TAG, "Camera at: $it")
+                    }
+                snapshotFlow { camera.location }
+                    .collect {
+                        Log.d(TAG, "Location at: $it")
+                    }
```

Comment:
I wouldn't expect this second collection to ever start, since the collection of the first will be suspending. Maybe `launch` these? Or just make two `LaunchedEffect`s and skip the `snapshotFlow`s

Reply:
Thanks! Going to use `launch` for both of these. I'd prefer to use `snapshotFlow` here since those values can change quite frequently

Reply:
Agreed